src/common/slate-types.ts

```diff
@@ -37,6 +37,22 @@ export enum EMetaFormat {
 export type EditorValue = Value;
 export type EditorContent = Document;
 
+export function isMarkFormat(format: EFormat) {
+  return [EFormat.bold, EFormat.italic, EFormat.underlined, EFormat.deleted,
+          EFormat.code, EFormat.superscript, EFormat.subscript, EFormat.color]
+          .includes(format);
+}
+
+export function isBlockFormat(format: EFormat) {
+  return [EFormat.heading1, EFormat.heading2, EFormat.heading3, EFormat.heading4, EFormat.heading5,
+          EFormat.heading6, EFormat.blockQuote, EFormat.listItem, EFormat.numberedList, EFormat.bulletedList]
+          .includes(format);
+}
+
+export function isInlineFormat(format: EFormat) {
+  return [EFormat.image, EFormat.link].includes(format);
+}
+
```

Comment:
I wrote these functions thinking I would need them for determining the enable/disable state of toolbar buttons by analyzing the contents of the current selection. I ended up not needing them but left them in on the grounds that they'll be useful at some point.